assets/src/edit-story/karma/fixture/fixture.js

```diff
@@ -205,6 +206,12 @@ export class Fixture {
   restore() {
     window.location.hash = '#';
     localStorage.clear();
+
+    // Set help center to closed right away
+    localStorage.setItem(
+      LOCAL_STORAGE_PREFIX.HELP_CENTER,
+      JSON.stringify({ isOpen: false, unreadTipsCount: 8 })
```

Comment:
Is the number important here? What if more tips are added in the future?

Reply:
8 is our current number of tips. it's not anything important. You're cool with setting local storage here? 

I'll swap the magic number out to pull from the actual length of the tips.

Reply:
No strong opinion on the localStorage usage here. Seems fine.